dluhc-component-library/README.md

```diff
@@ -1,47 +1,71 @@
-# dluhc-component-library
-
-This repository contains code for DLUHC components. These components are served via a CDN then can be rendered on any web page with JavaScript enabled.
-
-The components are written using Preact + Vite but compiled to a plain JS file for use within any frontend framework.
-
-# How to use
-
-The DLUHC components currently only support integration via direct JavaScript but in the future could be expanded to be directly included in React application.
-
-## Integrate with JavaScript
-
-DLUHC components are designed to be included into existing websites.
-
-Firstly you will need to include the dluhc-component-library JavaScript code in the head of your html. _src URL to be decided when CDN is chosen_
-
-```
-<head>
-	<script
-	      crossorigin="anonymous"
-	      src="https://cdn.jsdelivr.net/gh/digital-land/plan-making@release/assets/index-28d3d6bd.js"
-	    >
-	</script>
-</head>
-```
-
-Then where you want the component to be rendered you need to add a new div with a memorable id and a script tag rendering the component you want, referencing that id as shown below.
-
-```
-<body>
-	<div id="siteSelectionForm" role="main"></div>
-	<script>
-		window.DLUHC.renderSiteSelectionForm(
-			{},
-			document.getElementById("siteSelectionForm")
-		);
-	</script>
-</body>
-```
-
-# Components
-
-## Site Selection Form
-
-Basic information about the component can go here or link to a more detailed readme / story book for the component
-
-## Timetable
+# dluhc-component-library
+
+This repository contains code for DLUHC components. These components are served via a CDN then can be rendered on any web page with JavaScript enabled.
+
+The components are written using Preact + Vite but compiled to a plain JS file for use within any frontend framework.
+
+# How to use
+
+The DLUHC components currently only support integration via direct JavaScript but in the future could be expanded to be directly included in React application.
+
+## Integrate with JavaScript
+
+DLUHC components are designed to be included into existing websites.
+
+---

```

Comment:
Nice addition! Can we move the development section below the original instructions though please?

I think it might flow nicer having the core instructions then having a exception case for local development after as its primarily for LPAs to read.